docs/portfolio_analysis/attribute_table.md

```diff
@@ -64,16 +63,69 @@ general_df = df[[
 dp.DataTable(general_df)
 ```
 
-## Modeling
+## Turbine Modeling
+```{code-cell} ipython3
+:tags: [hide-input, full-width]
+
+turbine_df = df[[
+    "scope",
+    "system_location",
+    "time_domain",
+    "turbine_model",
+    "loads",
+    "design_optimization",
+    "costs",
+]]
+turbine_df = turbine_df[ turbine_df["scope" in ["turbine","any"]] ]
```

Comment:
`"scope" in ["turbine","any"]` is False so this turns into `turbine_df[ turbine_df[ False ] ]` and raises a KeyError. Maybe this was left over from debugging?

Reply:
I think the close-bracket just needs to move to be `turbine_df["scope"] in`...  I had trouble testing out some of these docs changes.  Can add that to our discussion agenda.